### DISCUSSION TITLE
What is the default value of maxAttempts for EmbedMolecule and EmbedMultipleConfs?

### DISCUSSION THREAD

**Discussion by YoavShamir5** [rdkit/rdkit#6541] [FAQ]
I am interested to know what is the default value for this argument in both cases, and also - how could I retrieve this value programmatically (not from documentation)?

Thanks!

**Comment by ptosco** [comment]
The default value is 10x the number of atoms as reported in the [C++ documentation](https://rdkit.org/docs/cppapi/namespaceRDKit_1_1DGeomHelpers.html#ac326b4681cdc0bfbb9978173d9bc4cab).
There is no way to retrieve that default programmatically, as it is hardcoded in the C++ code with no accessors:
https://github.com/rdkit/rdkit/blob/88c5d80b3309d2b2ae15d766434650029923fa05/Code/GraphMol/DistGeomHelpers/Embedder.cpp#L796
